sdk/servicebus/service-bus/package.json

```diff
@@ -87,24 +100,26 @@
     "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "build": "tsc -p . && rollup -c 2>&1",
+    "build:test:browser": "tsc -p . && cross-env BROWSER_TEST=true rollup -c 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test": "npm run build:test:node && npm run build:test:browser",
+    "build": "npm run build:node && npm run build:browser && npm run build:test:node && npm run build:test:browser",
     "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-esm test-dist typings *.tgz *.log",
     "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "echo skipped",
+    "integration-test:browser": "karma start --single-run",
     "integration-test:node": "mocha -t 120000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "tslint -p . -c tslint.json --fix",
     "lint": "tslint -p . -c tslint.json",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
     "preinstall": "node ../../../common/scripts/rush-welcome.js",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "test:browser": "npm run integration-test:browser",
+    "test:node": "npm run integration-test:node",
```

Comment:
You might want to keep the `npm run build:test &&` part on both test:node and test:browser. Developers seem to expect that.

Reply:
Same comment as above.
Is this something that's going to be done for all other package.json files moving forward?

Just an input from my end - I found the separation easier for when I needed to troubleshoot a particular build and simply re-run tests without building again.

Reply:
I would keep the build and test scripts as they were. Most developers won't have to deal with figuring out build related issues.

Reply:
Got it, please take a look at the updated scripts.

Reply:
About reverting to `"build": "tsc -p . && rollup -c 2>&1",` I'm a little doubtful as the build steps for node and browser seem different?

Reply:
I think that the output from the compiler to be the same in both node and browser builds. The rollup process is what will bundle for the different platforms. 

What happens when you revert it? Does that work or does that cause other problems?

/cc @bterlson

Reply:
It's currently reverted and ran fine on the CI/CD without any problems as such.